### DISCUSSION TITLE
❓Support: E1696 cannot open source file "onnxruntime_cxx_api.h" silero_vad

### DISCUSSION THREAD

**Discussion by ITHealer** [snakers4/silero-vad#404] [Q&A]
I am trying to run source cpp (wav.h and silero-vad-onnx.cpp on VS studio) in example. 
However I cannot access the onnxruntime_cxx_api.h lib. Please help me with this?

**Comment by snakers4** [comment]
Maybe it needs to be compiled?

**Reply by ITHealer** [reply]
![image](https://github.com/snakers4/silero-vad/assets/94621494/a34edb49-f966-4875-87cd-3028cecdf818)

What do I do next to compile it?
